packages/app-project/components/ClassifyPage/components/FinishedForTheDay/FinishedForTheDay.js

```diff
@@ -0,0 +1,56 @@
+import counterpart from 'counterpart'
+import { Box, Button, Grid, Heading, Paragraph } from 'grommet'
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+
+import en from './locales/en'
+import ProjectImage from './components/ProjectImage'
+import ClassifyBox from '../ClassifyBox'
+
+counterpart.registerTranslations('en', en)
+
+const StyledButton = styled(Button)`
+  border-width: 1px;
```

Comment:
We might want to set a default border width in the theme for buttons if we're finding ourselves modifying this a lot.